components/raindrop/raindrop.app.mjs

```diff
@@ -1,11 +1,167 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "raindrop",
-  propDefinitions: {},
+  propDefinitions: {
+    raindropID: {
+      type: "string",
+      label: "Raindrop ID",
+      description: "Existing raindrop ID",
+    },
```

Comment:
Set the label and description to say Bookmark instead of Raindrop so they don't need to be overwritten in actions.
Also, could we use the method `getRaindrops()` to create async options for this prop?
```suggestion
    raindropID: {
      type: "string",
      label: "Bookmark ID",
      description: "Existing bookmark ID",
    },
```